### PR TITLE
[WIP] ar.Iproj BY MTR .. update

### PR DESCRIPTION
**Pull request by MTR7**

### CODE REVIEW

**Comment by MTR7** [comment]
new file update will be upload during 48H

**Comment by MTR7** [comment]
nonce

File: [WIP] ar.Iproj BY MTR [hunk 1]
```diff
@@ -10,18 +10,18 @@
...
 "Ready to jailbreak" = "جاهز لي تفعيل جيلبريك"
 "Error" = "خطاء"
 "Bundled Resources version is missing. This build is invalid." = "إصدار الموارد للنسخة مفقود. هذا البناء غير صالح."
 "Days" = "أيام"
 "Invalid Entry" = "معطيات خاطئة"
-"The boot nonce entered could not be parsed" = "لا يمكن تحليل التمهيد الذي أدخلته"
+"The boot nonce entered could not be parsed" = "لا يمكن تحليل نص الآدخال الذي أدخلته"
 "OK" = "OK"
 "The device will be restarted." = "سيتم إعادة تشغيل الجهاز"
-"Copy boot nonce?" = "نسخة تمهيدية للنوس؟"
```
Comment (MTR7) [inline [WIP] ar.Iproj BY MTR]: نسخة تمهيدية لنص الادخال (nonce)

**Comment by Samgisaninja** [comment]
Thanks for your assistance with translating unc0ver, however I am closing this pull because:

* Please create your own .lproj folder, do not modify files outside your directory unless you are improving another translation

**Comment by iA7myd** [comment]
Hi Samgisaninja, you don't need to accept any Arabic localization (ar) any more, I'm already finished this language.

**Pull request closed without merging**